include/ignition/common/SystemPaths.hh

```diff
@@ -34,9 +34,10 @@
 #include <vector>
```

Comment:
Here too remove `memory`? The same goes for various other files.

Reply:
4edd805